glean-core/rlb/src/lib.rs

```diff
@@ -89,6 +89,10 @@ impl glean_core::OnGleanEvents for GleanEvents {
         // intentionally left empty
         Ok(())
     }
+
+    fn on_shutdown(&self) {
```

Comment:
Right now:
2 with `on_`: `on_initialize_finished`, `on_shutdown`,
3 without: `trigger_upload`, `start_metrics_ping_scheduler`, `cancel_uploads`

Without `on_` definitely sounds right for the 3 that don't have it. `shutdown` instead of `on_shutdown` sounds ok to me as well.
`initialize_finished` works as well.

So ... going with no `on_` (Heh, it's in the trait name already anyway)